src/Components/AuthDialog/AuthDialogLeftPanel.tsx

```diff
@@ -0,0 +1,195 @@
+import {
+  Box,
+  Carousel,
+  CarouselCell,
+  CarouselCellProps,
+  CarouselRail,
+  CarouselRailProps,
+  Flex,
+  Image,
+} from "@artsy/palette"
+import { useFlag } from "@unleash/proxy-client-react"
+import {
+  FC,
+  forwardRef,
+  ForwardRefExoticComponent,
+  useEffect,
+  useRef,
+} from "react"
+import styled from "styled-components"
+import { useCursor } from "use-cursor"
+import { resized } from "Utils/resized"
+
+const MODAL_WIDTH = 900
+
+export const AuthDialogLeftPanel: FC<React.PropsWithChildren> = () => {
+  const img = resized(IMAGE.src, { width: MODAL_WIDTH / 2 })
+
+  const newSignupEnabled = useFlag("onyx_new-signup-modal")
+
+  if (!newSignupEnabled) {
+    return (
+      <Box display={["none", "block"]} width="100%">
+        <Image
+          {...img}
+          width="100%"
+          height="100%"
+          lazyLoad
+          alt=""
+          style={{ objectFit: "cover" }}
+        />
+      </Box>
+    )
+  }
+
+  return (
+    <Box display={["none", "block"]} width="100%" overflow="hidden">
+      <ImageSlider />
+    </Box>
+  )
+}
+
+const ImageSlider: FC = () => {
+  const intervalRef = useRef<ReturnType<typeof setInterval> | null>(null)
+
+  const { index, setCursor } = useCursor({ max: DEFAULT_IMAGES.length })
+
+  const stopAutoPlay = () => {
+    intervalRef.current && clearInterval(intervalRef.current)
+  }
+
+  useEffect(() => {
+    intervalRef.current = setInterval(
+      () => setCursor(prevCursor => prevCursor + 1),
+      5000,
+    )
+
+    return stopAutoPlay
+  }, [setCursor])
+
+  const handleClick = (index: number) => {
+    setCursor(index)
+    stopAutoPlay()
+  }
+
+  return (
+    <Flex width={COLUMN_WIDTH} height="100%" position="relative">
```

Comment:
Isn't position="relative" default value? 🤔 If so, we could remove it from the props

Reply:
`static` is default